pb/profile.pb.go

```diff
@@ -28,33 +28,29 @@ var _ = math.Inf
-	SecondaryColor     string                     `protobuf:"bytes,17,opt,name=secondaryColor" json:"secondaryColor,omitempty"`
-	TextColor          string                     `protobuf:"bytes,18,opt,name=textColor" json:"textColor,omitempty"`
-	HighlightColor     string                     `protobuf:"bytes,19,opt,name=highlightColor" json:"highlightColor,omitempty"`
-	HighlightTextColor string                     `protobuf:"bytes,20,opt,name=highlightTextColor" json:"highlightTextColor,omitempty"`
-	AvatarHashes       *Profile_Image             `protobuf:"bytes,21,opt,name=avatarHashes" json:"avatarHashes,omitempty"`
-	HeaderHashes       *Profile_Image             `protobuf:"bytes,22,opt,name=headerHashes" json:"headerHashes,omitempty"`
-	FollowerCount      uint32                     `protobuf:"varint,23,opt,name=followerCount" json:"followerCount,omitempty"`
-	FollowingCount     uint32                     `protobuf:"varint,24,opt,name=followingCount" json:"followingCount,omitempty"`
-	ListingCount       uint32                     `protobuf:"varint,25,opt,name=listingCount" json:"listingCount,omitempty"`
-	BitcoinPubkey      string                     `protobuf:"bytes,26,opt,name=bitcoinPubkey" json:"bitcoinPubkey,omitempty"`
-	LastModified       *google_protobuf.Timestamp `protobuf:"bytes,27,opt,name=lastModified" json:"lastModified,omitempty"`
+	Handle           string                   `protobuf:"bytes,1,opt,name=handle" json:"handle,omitempty"`
+	Name             string                   `protobuf:"bytes,2,opt,name=name" json:"name,omitempty"`
+	Location         string                   `protobuf:"bytes,3,opt,name=location" json:"location,omitempty"`
+	About            string                   `protobuf:"bytes,4,opt,name=about" json:"about,omitempty"`
+	ShortDescription string                   `protobuf:"bytes,5,opt,name=shortDescription" json:"shortDescription,omitempty"`
+	Website          string                   `protobuf:"bytes,6,opt,name=website" json:"website,omitempty"`
+	Email            string                   `protobuf:"bytes,7,opt,name=email" json:"email,omitempty"`
+	PhoneNumber      string                   `protobuf:"bytes,8,opt,name=phoneNumber" json:"phoneNumber,omitempty"`
+	Social           []*Profile_SocialAccount `protobuf:"bytes,9,rep,name=social" json:"social,omitempty"`
+	Nsfw             bool                     `protobuf:"varint,10,opt,name=nsfw" json:"nsfw,omitempty"`
+	Vendor           bool                     `protobuf:"varint,11,opt,name=vendor" json:"vendor,omitempty"`
+	Moderator        bool                     `protobuf:"varint,12,opt,name=moderator" json:"moderator,omitempty"`
+	ModInfo          *Moderator               `protobuf:"bytes,13,opt,name=modInfo" json:"modInfo,omitempty"`
+	Colors           *Profile_Colors          `protobuf:"bytes,14,opt,name=colors" json:"colors,omitempty"`
+	Stats            *Profile_Stats           `protobuf:"bytes,15,opt,name=stats" json:"stats,omitempty"`
```

Comment:
the cool thing about encapsulating these stats in a struct, is that now you can think of keeping a history of them if you add a `LastModified` field on the `Profile_Stats` struct.

And then in a future upgrade, we could see how a profile has behaved during a certain period of time by having here instead

`Stats   []*Profile_Stats`